### PR TITLE
EZ-393 Integrating faq, map, quote and social media components

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

File: pattern-lab/source/_patterns/00-atoms/02-text/01-paragraph.twig [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 <p >
   {% block paragraph_content %}
-    {{ paragraph_content }}
+    {{ paragraph_content|striptags }}
```
Comment (kuldev) [inline pattern-lab/source/_patterns/00-atoms/02-text/01-paragraph.twig:19]: a paragraph can have tags eg: a, strong etc. so this striptags should not be here.

File: templates/paragraphs/paragraph--map.html.twig [hunk 2]
```diff
@@ -50,22 +50,23 @@
...
 {% set data = {
   map:{
-    title:"Map Component",
-    subTitle:" India Taj Mahal is an exceptional beauty",
-    location:" Taj Mahal",
-    url:"https://maps.google.com/maps?hl=en&q=Agra&t=m&z=14&output=embed",
-    body:"Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Maecenas faucibus mollis interdum. Cras mattis consectetur purus sit amet fermentum. Vestibulum id ligula porta felis euismod semper. Nullam id dolor id nibh ultricies vehicula ut id elit. Donec ullamcorper nulla non metus auctor fringilla. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Nullam quis risus eget urna mollis ornare vel eu leo. Curabitur blandit tempus porttitor",
+    title:paragraph.field_title.value,
+    subTitle:paragraph.field_short_title.value,
+    url:paragraph.field_map.value,
```
Comment (kuldev) [inline templates/paragraphs/paragraph--map.html.twig:55]: add more spacing around ':'

File: templates/paragraphs/paragraph--social-media.html.twig [hunk 3]
```diff
@@ -47,10 +47,21 @@
...
+  }
+} %}
+
 {% block paragraph %}
   <div{{ attributes.addClass(classes) }}>
+    {{ content.contextual_links }}
     {% block content %}
-      {{ content }}
+      {% embed "@molecules/04-socialMedia/00-social-media.twig" with data %}
+      {% endembed %}
```
Comment (kuldev) [inline templates/paragraphs/paragraph--social-media.html.twig:64]: 'endembed' or 'embed' ?

**Review by kuldev — COMMENTED** [review]
please run some code linting, for spacing and names.

File: pattern-lab/source/_patterns/00-atoms/02-text/01-paragraph.twig (hunk 1 above)
Comment (surbhi90) [reply pattern-lab/source/_patterns/00-atoms/02-text/01-paragraph.twig]: Removed it

File: templates/paragraphs/paragraph--map.html.twig (hunk 2 above)
Comment (surbhi90) [reply templates/paragraphs/paragraph--map.html.twig]: Done

File: templates/paragraphs/paragraph--social-media.html.twig (hunk 3 above)
Comment (surbhi90) [reply templates/paragraphs/paragraph--social-media.html.twig]: It is 'endembed' but not instead of it used include

**Comment by kuldev** [comment]
@vaibhavjain-in looks fine to me, could you please merge it.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**